gtsam/base/Testable.h

```diff
@@ -41,6 +41,7 @@
 #include <string>
 
 #define GTSAM_PRINT(x)((x).print(#x))
+#define GTSAM_COUT(x) (std::cout << #x ": " << x << std::endl)
```

Comment:
But print already assumes cout

Reply:
This is more for types that don't support `.print()`. I can use this to print doubles, strings etc.

Reply:
Wouldn't it be convenient to do `GTSAM_COUT(x)` instead of `std::cout << "x: " << x << std::endl;`?

Reply:
Not a GTSAM thing

Reply:
It's not a GTSAM thing, but more of a helper thing to make debugging and analysis easier. Oh well.